src/qutip_tensorflow/__init__.py

```diff
@@ -1 +1,2 @@
 from .version import version as __version__
+from .core import *
```

Comment:
Do you need a separate `core` package within `qutip_tensorflow`?  I wouldn't imagine there'll be too many different levels in it, so this may just be adding needless structure.

Reply:
I was following the same structure that QuTiP has for consistency. It is no necessary but I though someone working on QuTiP would find it more appealing.

Reply:
I'd say there's no need to try and maintain structural consistency between separate packages. QuTiP's core is just for code organisation anyway - it's not directly exposed to users.